### PR TITLE
Make sure the press releases render

### PR DESCRIPTION
**Pull request by davidpmccormick**
☝️ 

Adding a test to make sure the asyncSearchResults puts something on the page (after #9391)

### CODE REVIEW

File: content/webapp/components/SearchResults/SearchResults.tsx [hunk 1]
```diff
@@ -49,7 +49,10 @@ const SearchResults: FunctionComponent<Props> = ({
     {items.map(
       (item, index) =>
         item.type !== 'card' && (
-          <Result key={item.id}>
+          <Result
+            key={item.id}
+            data-testid={index === 0 ? 'search-result' : undefined}
```
Comment (rcantin-w) [inline content/webapp/components/SearchResults/SearchResults.tsx:54]: I thought there was a typo here but noticing now we have both `data-testid` (although only once) and `data-test-id`. Doesn't really matter but we could align?

Comment (rcantin-w) [reply content/webapp/components/SearchResults/SearchResults.tsx:54]: I didn't know about `getByTestId`! Sounds like a nice thing to make use of!

**Review by rcantin-w — APPROVED** [review]

File: content/webapp/components/SearchResults/SearchResults.tsx (hunk 1 above)
Comment (davidpmccormick) [reply content/webapp/components/SearchResults/SearchResults.tsx:54]: I started with `data-test-id` but I didn't think Playwright was picking up on it. It's possible I had that wrong though and something else was preventing the test from passing before – let me check. Seems like this is the [default for Playwright](https://playwright.dev/docs/locators#set-a-custom-test-id-attribute) but it's configurable

Comment (rcantin-w) [reply content/webapp/components/SearchResults/SearchResults.tsx:54]: I'm not against going with the default tbh, but am thinking we should defo make use of the helper function across! TIL 😁

Comment (davidpmccormick) [reply content/webapp/components/SearchResults/SearchResults.tsx]: I think the philosophy is that using `data-test-id` should be a last resort and preferably you'd select by something a user can actually see. But sometimes you need it as an escape hatch

Comment (rcantin-w) [reply content/webapp/components/SearchResults/SearchResults.tsx]: Oh absolutely agree! I meant where we do lookups like `.find("[data-test-id='TextInputErrorMessage']"))`, we could look at using `getByTestId` instead, just didn't know we could :)

Comment (rcantin-w) [reply content/webapp/components/SearchResults/SearchResults.tsx:54]: thanks for trying! but yeah don't let that block this, I shared with Adi too and we'll aim to use that moving forward!

**Pull request merged**